driver/pace/driver/run.py

```diff
@@ -577,7 +659,9 @@ def main(driver_config: DriverConfig, comm):
         config=driver_config,
         comm=comm,
     )
-    driver.step_all()
+    with driver.performance_config.total_timer.clock("total"):
+        driver.step_all()
```

Comment:
nit: if this is purely timing `step_all` I think you can move it inside that routine. This would avoid having a doubly-nested access (driver.<object>.<sub-object>.<method>). It only needed to be at a higher level earlier when you were timing across methods.